components/product-details/index.js

```diff
@@ -1,24 +1,33 @@
+import NavBar from '../NavBar';
+import personData from '../../mock/person.json';
 import PropTypes from 'prop-types';
 import path from 'path';
+import Image from 'next/image';
 
 export const ProductDetails = ({ productJSON }) => {
   return (
-    <div className="product-container">
-      <img
-        className="product-image"
-        src={path.join('../assets', productJSON.image)}
-        alt={productJSON.id}
-      />
-      <div className="product-details">
-        <h2> {productJSON.name}</h2>
-        <h3> Where to use</h3>
-        <ul>
-          {productJSON.usage.map((use, index) => {
-            return <li key={index}>{use}</li>;
-          })}
-        </ul>
-      </div>
-      <style jsx>{`
+    <>
+      <NavBar personData={personData} />
+      <div className="product-container">
+        <div className="product-image">
+          <Image
+            src={path.join('/assets', productJSON.image)}
+            alt={productJSON.id}
+            width={200}
+            height={200}
+          />
+        </div>
+
+        <div className="product-details">
+          <h2> {productJSON.name}</h2>
+          <h3> Where to use</h3>
+          <ul>
+            {productJSON.usage.map((use, index) => {
+              return <li key={index}>{use}</li>;
```

Comment:
Using `index` as the key is not a good practise. Are we not getting any product id from the `productJSON` ?